protobuf/internal/proto_repositories.bzl

```diff
@@ -6,11 +6,14 @@ def proto_repositories(excludes = [],
                        lang_requires = [],
                        protobuf_deps = DEPS,
                        protobuf_requires = [
+                         "bazel_skylib",
                          "com_google_protobuf",
                          "protobuf",
                          "protobuf_clib",
                          "protocol_compiler",
                          "protobuf_headers",
+                         "zlib",
+                         "com_github_madler_zlib",
```

Comment:
Did zlib end up being required for all the language implementations with 3.7.1?

Reply:
zip ended up being a protobuf dep instead of a grpc one i think